test/cluster.js

```diff
@@ -157,6 +157,20 @@ describe('Bigtable/Cluster', function() {
       cluster.create(options, done);
     });
 
+    it('should accept fully expanded name as clusterId', function() {
+      let id_ = `${INSTANCE.name}/clusters/${CLUSTER_ID}`;
```

Comment:
shouldn't this just be `id =`?  Do we need the `_`?

Reply:
Can't we just use CLUSTER_NAME instead of _id?  Also, why not use `var` like everywhere else?

Reply:
done